test/server/cards/02_SHD/units/RicketyQuadjumper.spec.ts

```diff
@@ -0,0 +1,90 @@
+describe('RicketyQuadjumper', function () {
+    integration(function (contextRef) {
+        describe('Rickety Quadjumper\'s ability', function () {
+            it('should give an experience token to another unit if the revealed card is not a unit', function () {
+                const { context } = contextRef;
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['wampa', 'battlefield-marine'],
+                        spaceArena: ['rickety-quadjumper'],
+                        deck: ['protector'],
+                    },
+                    player2: {
+                        groundArena: ['atst'],
+                    }
+                });
+
+
```

Comment:
nit: should only be one newline, not two. handful of cases of this throughout the file